paper-server/src/main/java/io/papermc/paper/command/brigadier/argument/operation/OperationImpl.java

```diff
@@ -0,0 +1,63 @@
+package io.papermc.paper.command.brigadier.argument.operation;
+
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import net.minecraft.commands.arguments.OperationArgument;
+import net.minecraft.world.scores.ScoreAccess;
+import net.minecraft.world.scores.ScoreHolder;
+import org.bukkit.craftbukkit.scoreboard.CraftObjective;
+import org.bukkit.craftbukkit.scoreboard.CraftScoreHolder;
+import org.bukkit.craftbukkit.scoreboard.CraftScoreboard;
+import org.bukkit.scoreboard.Objective;
+import org.bukkit.scoreboard.Scoreboard;
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+
+@NullMarked
+public class OperationImpl implements Operation {
```

Comment:
All the downsides mentioned in the API type apply here but review of this without changes to the API type is pretty useless.